### ISSUE TITLE
NoSuchMethodException on API 10

### ISSUE THREAD

**Issue by rubengees** [mikepenz/AboutLibraries#195]
I get the following crash when trying to show the AboutLibraries Activity at API 10:

```
Caused by: java.lang.NoSuchMethodException
    at java.lang.Class.getDeclaredMethods(Native Method)
    at java.lang.ClassCache.getDeclaredMethods(ClassCache.java:140)
    at java.lang.Class.getDeclaredMethods(Class.java:757)
    at java.io.ObjectStreamClass.computeSerialVersionUID(ObjectStreamClass.java:554)
    at java.io.ObjectStreamClass.createClassDesc(ObjectStreamClass.java:252)
    at java.io.ObjectStreamClass.lookupStreamClass(ObjectStreamClass.java:982)
    at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1747)
    at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1689)
    at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1653)
    at android.os.Parcel.writeSerializable(Parcel.java:1171)
    at android.os.Parcel.writeValue(Parcel.java:1130)
    at android.os.Parcel.writeMapInternal(Parcel.java:488)
    at android.os.Bundle.writeToParcel(Bundle.java:1552)
    at android.os.Parcel.writeBundle(Parcel.java:502)
    at android.content.Intent.writeToParcel(Intent.java:5474)
    at android.app.ActivityManagerProxy.startActivity(ActivityManagerNative.java:1341)
    at android.app.Instrumentation.execStartActivity(Instrumentation.java:1374)
    at android.app.Activity.startActivityForResult(Activity.java:2827)
    at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:843)
    at android.app.Activity.startActivity(Activity.java:2933)
    at com.mikepenz.aboutlibraries.LibsBuilder.start(LibsBuilder.java:520)
```

Above the warning:

```
Could not find class 'com.mikepenz.aboutlibraries.ui.LibsFragment', 
referenced from method com.mikepenz.aboutlibraries.LibsBuilder.fragment
```

This was tested on a standard emulator. It doesn't happen on an emulator with API level 16.  

This is how I start the Activity:

```
new LibsBuilder().withActivityTitle(getString(R.string.app_about_title))
                        .withAboutAppName(getString(R.string.app_name))
                        .withAboutDescription(getString(R.string.app_about))
                        .withAboutIconShown(true).withAboutVersionShownName(true)
                        .withActivityStyle(Libs.ActivityStyle.LIGHT_DARK_TOOLBAR)
                        .withAutoDetect(false).withLibraries("butterknife", "mpandroidchart")
                        .withFields(R.string.class.getFields())
                        .start(this);
```

No Proguard involved.

**Comment by rubengees** [comment]
Oops sorry about the duplicate...

**Comment by mikepenz** [comment]
fixed in v5.6.0
